COMPONENT_STATUS.md

```diff
@@ -5,68 +5,68 @@ part of Canvas.
 
 :white_check_mark: - Available
 
-:microscope: - [Labs Components](modules/_labs)
+:microscope: - [Labs Components](./modules/_labs)
```

Comment:
I'm changing the URL pattern to signify relative links for the webpack loader to pick up.